### PR TITLE
[webview_flutter] General improvements

### PR DESCRIPTION
**Pull request by swift-kim**
- Change the project type to staticLib.
- Enable back key navigation.
- Implement `WebSettings.javascriptMode` and `WebViewController.clearCache`.
- Redirect console messages to stdout/stderr for easy debugging.
- Minor code cleanups.

### CODE REVIEW

File: packages/webview_flutter/tizen/src/webview.cc [hunk 1]
```diff
@@ -190,14 +197,18 @@ void WebView::ApplySettings(const flutter::EncodableMap& settings) {
           has_progress_tracking_ = std::get<bool>(value);
         }
       } else if (string_key == "debuggingEnabled") {
+        // Not supported on Tizen.
       } else if (string_key == "gestureNavigationEnabled") {
+        // Not implemented.
       } else if (string_key == "allowsInlineMediaPlayback") {
+        // Not applicable for Tizen (always allowed).
```
Comment (swift-kim) [inline packages/webview_flutter/tizen/src/webview.cc:204]: This is true for the lightweight web engine. Is this also true for EWK?

File: packages/webview_flutter/tizen/src/webview.cc [hunk 2]
```diff
@@ -339,23 +356,20 @@ void WebView::InitWebView() {
...
-  if (manager) {
+  Ewk_Cookie_Manager* cookie_manager = ewk_context_cookie_manager_get(context);
+  if (cookie_manager) {
     ewk_cookie_manager_accept_policy_set(
-        manager, EWK_COOKIE_ACCEPT_POLICY_NO_THIRD_PARTY);
+        cookie_manager, EWK_COOKIE_ACCEPT_POLICY_NO_THIRD_PARTY);
   }
-  EwkInternalApiBinding::GetInstance().settings.ImePanelEnabledSet(settings,
-                                                                   true);
-  ewk_settings_javascript_enabled_set(settings, true);
```
Comment (JSUYA) [inline packages/webview_flutter/tizen/src/webview.cc:352]: Is javascript enabled still true after deleting this line?

**Review by JSUYA — COMMENTED** [review]
Shouldn't `int color`(in 150 line) be changed to int32_t as well?

File: packages/webview_flutter/tizen/src/webview.cc (hunk 2 above)
Comment (swift-kim) [reply packages/webview_flutter/tizen/src/webview.cc:352]: Yes, and actually it's not deleted. `ApplySettings` is always called during the webview initialization and `ewk_settings_javascript_enabled_set` is called there.

**Comment by swift-kim** [comment]
@JSUYA Either way should be okay but `int32_t` seems to make more sense since the value is expected to be exactly 32-bits long.

**Review by JSUYA — APPROVED** [review]

File: packages/webview_flutter/tizen/src/webview.cc (hunk 1 above)
Comment (bbrto21) [reply packages/webview_flutter/tizen/src/webview.cc:204]: I think this feature only for iOS. and as far as I know, most weview implementations are allowed inline media playback by default. so I think you don't have to care about it.

**Review by bbrto21 — APPROVED** [review]
LGTM

**Pull request merged**